packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/DataVisAnnotationTask/DataVisAnnotationTask.js

```diff
@@ -0,0 +1,96 @@
+import counterpart from 'counterpart'
+import { Text } from 'grommet'
+import { inject, observer, PropTypes as MobXPropTypes } from 'mobx-react'
+import { observable } from 'mobx'
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Markdownz } from '@zooniverse/react-components'
+import TaskInputField from '../TaskInputField'
+import InputStatus from '../InputStatus'
+import InputIcon from '../InputIcon'
+import Graph2dRangeXIcon from './components/Graph2dRangeXIcon'
+
+import en from './locales/en'
+counterpart.registerTranslations('en', en)
+
+export const StyledFieldset = styled.fieldset`
+  border: none;
+`
+
+function storeMapper (stores) {
+  const annotations = stores.classifierStore.classifications.currentAnnotations
+  const { active, setActive } = stores.classifierStore.dataVisAnnotating
+  return {
+    active,
+    annotations,
+    setActive
+  }
+}
+
+@inject(storeMapper)
+@observer
+class DataVisAnnotationTask extends React.Component {
+  onChange(index, event) {
+    if (event.target.checked) {
+      this.props.setActive(index)
+    }
+  }
+
+  render () {
+    const {
+      active,
+      annotations,
+      task
+    } = this.props
+    let annotation
+    if (annotations && annotations.size > 0) {
+      annotation = annotations.get(task.taskKey)
+    }
+    // TODO: Add in the status count along with the validations to not go over the max allowed
+    const numberOfAnnotations = (annotation && annotation.value.length) || 0
+
+    return (
+      <StyledFieldset>
+        <Text size='small' tag='legend'>
+          <Markdownz>
+            {task.instruction}
+          </Markdownz>
+        </Text>
+        
+        {task.tools.map((tool, index) => {
+          const checked =  active === index
+          return (
+            <TaskInputField
+              checked={checked}
+              index={index}
+              key={`${task.taskKey}_${index}`}
+              label={tool.label}
+              labelIcon={<InputIcon icon={<Graph2dRangeXIcon />} />}
```

Comment:
Getting `React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: object.` related to one of these components.

Reply:
I might have screwed up a renaming of something

Reply:
I've fixed this and also the export of InputIcon.